test/test_unstructured_2d.jl

```diff
@@ -420,7 +416,7 @@ end
                             0.17625472321992852,
                             2.6407324614341476e-5,
                         ],
-                        surface_flux=(FluxHydrostaticReconstruction(flux_hll,
+                        surface_flux=(FluxHydrostaticReconstruction(FluxHLL(min_max_speed_naive),
```

Comment:
I'm not sure whether we really want to test the naive version here or the Davis version.